package.json

```diff
@@ -15,6 +15,7 @@
     "test": "ng test routerkit",
     "lint": "ng lint routerkit",
     "parse": "schematics ./dist/routerkit/collection.json:parse --project example-app",
+    "chore:restore-package-json": "git clean -f package-lock.json && git checkout package.json",
```

Comment:
what makes package-lock.json generate?
If, nevertheless, something calls, I suggest to insert this script without alias into tasks.json right away, because this script has nothing to do with the project since we use primarily use

Reply:
It generates something like

```json
{
 @routerkit/core": "file:dist/routerkit"
}
```

after adding the schematic like

```bash
ng add ./dist/routerkit/ --skipInstall
```

Alright, I'll remove the script and execute the task as a shell command.